docs/static-websites.md

```diff
@@ -76,5 +76,34 @@ mkdir -p less/statics/demo-website
   
 </Tabs>
 
-## Deploy
+## Custom Domain
+Less also allows you to use a custom domain with your static websites. Use the following command to set up a custom domain for a specific static site:
+```bash
+npx @chuva.io/less-cli domains --project-name PROJECT_NAME --static-name STATIC_NAME --custom-domain CUSTOM_DOMAIN
+```
+
+For example, if we want to use a custom domain with `demo-website` the command will be the following
```

Comment:
Can this be :
For example, if we want to use a custom domain 'demo-website.com' for the 'demo-website' project, the command will be the following: